### ISSUE TITLE
Filter widget performance on IE9 for table with 1000 rows

### ISSUE THREAD

**Issue by rolinex** [Mottie/tablesorter#645]
I am using the filter widget, my table has 1000 rows. On IE9, it takes almost 2 extra seconds to display the table when I use the filter widget.  This is because the filter widget initialisation takes approx 2 seconds ( from console log: Completed filter widget search (1609ms) ). I found that the extra 2 seconds are from findRows() which calls ts.processTbody() twice, each call taking over half a second ($tb.detach(); and $tb.insertAfter(holdr); taking most of that time).  Is there a way to avoid calling findRows() or processTbody() when the widget is being initialised to reduce the time it takes to display the table?  I am using version 2.14.5.
Chrome and IE11 is much better but I am stuck with IE9...
